testnets/doravotaincentivetestnet/chain.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "doravotaincentivetestnet",
```

Comment:
can we please change this to `"chain_name": "doravotatestnet2"`?

Reply:
good idea. I'm already changed